tools/wt_verify.py

```diff
@@ -148,11 +159,70 @@ def parse_output():
                 is_root_node = False
     return output
 
-def visualize(data, visualization_type):
-    """
-    Visualizing data
-    """
-    pass
+
+def histogram(field, chkpt, chkpt_name):
+    internal = []
+    leaf = []
+    for val in chkpt.values():
+        if field in val:
+            if "internal" in val["page_type"]:
+                internal.append(val[field])
+            elif "leaf" in val["page_type"]:
+                leaf.append(val[field])
+    fig, (ax1, ax2) = plt.subplots(2, figsize=(15, 10))
+    ax = (ax1, ax2)
```

Comment:
Is there something that prevents us from just doing?
```suggestion
    fig, ax = plt.subplots(2, figsize=(15, 10))
```
?

Reply:
fixed https://github.com/wiredtiger/wiredtiger/pull/10196/commits/297743d117f7f86aac38e6e2e9c3db15e6fdaef8